File: Data Structures & Algorithms/rotate-array/submission-2.py
class Solution:
    def rotate(self, nums: List[int], k: int) -> None:
        newList = []
        for i in range(len(nums)):
            newList.append(nums[(i-k) % (len(nums))])
        for i in range(len(newList)):
            nums[i] = newList[i]

File: Data Structures & Algorithms/concatenation-of-array/submission-2.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> newList;
        for (int i=0;i<2*nums.size();i++){
            newList.push_back(nums.at(i%nums.size()));
        }
        return newList;
    }
};